### ISSUE TITLE
The progress-dot's position is not right

### ISSUE THREAD

**Issue by SongJuXin** [videojs/video.js#7004]
## Description
The progress-dot's position is not right,and the same phenomenon is occurs in volume-bar. 
![image](https://user-images.githubusercontent.com/20874344/102781500-bc85a380-43d2-11eb-84e8-7288b4e7f96f.png)

My archive on the base of videojs as follows.
![image](https://user-images.githubusercontent.com/20874344/102780051-3e280200-43d0-11eb-9a15-140ada1159d0.png)
https://codepen.io/SongJuXin/pen/WNGEMRX is the online show.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1.play video
2.observe progress-dot 

### Expected
 Progress-dot  is vertical middle.

### Actual
Progress-dot  is dropdown.

### resolve method
reason:`\f111` character has white gap.
resolve method:update the  font file is used  of the section.
 ```css
.vjs-icon-circle:before, .video-js .vjs-mouse-display:before, .video-js .vjs-play-progress:before, .video-js .vjs-volume-level:before {
    content: "\f111";
}
```

**Comment by gkatsev** [comment]
Looks like you're testing with Video.js 5 which is a really old version at this point. Latest Video.js does not seem to have this problem since the dot is made purely using CSS, without any font icons.
I would recommend upgrading because there have been many improves since the last v5 release, almost 3 years ago.

**Comment by SongJuXin** [comment]
@gkatsev Thanks reply.
Now,I replace with the lastest v7.10.2.
![image](https://user-images.githubusercontent.com/20874344/102845770-3bb5be80-4449-11eb-8f5d-7ed3c7ca07b1.png)
The same problem occured again.
![image](https://user-images.githubusercontent.com/20874344/102845934-a6ff9080-4449-11eb-9cb7-4d82ec996b46.png)

**Comment by gkatsev** [comment]
The way you create the large progress bar is not equivalent to how it is done currently.
If we set the player's font-size really large you can see that it is nicely centered on top of the progress control https://codepen.io/gkatsev/pen/wvzrdxq

**Comment by SongJuXin** [comment]
Ok,my previous achieve have some flaw.But the small size control actully has position problem.
This is origin image.
![normal](https://user-images.githubusercontent.com/20874344/102962731-69bb0180-4522-11eb-9785-f32addbfd27f.png)
This is scale 10x image.
![big](https://user-images.githubusercontent.com/20874344/102962737-6b84c500-4522-11eb-9c85-fb70c14d140b.png)
We can see the dot is dropdown when the size is normal.Big size has no problem.

**Comment by gkatsev** [comment]
I don't see it. Zooming in on the control bar that is included with latest Video.js and it's still centered.

**Comment by SongJuXin** [comment]
Merry Chrismas!
I maked a website to show canvas.I fade the progress-dot.This is the website content:
![image](https://user-images.githubusercontent.com/20874344/103264579-0fb8b100-49e6-11eb-8177-71810e10a1c6.png)

https://songjuxin.github.io/videojs-exmpale/video-case.html is the url.
I think the issue phenomenon is clear now.

**Comment by mister-ben** [comment]
The dot is in the correct position for me on your example page (and the earlier links). it looks like Chrome, but which browser are you using? Does it happen in incognito? I'm wondering if you have an extension inserting conflicting styles or similar.

**Comment by SongJuXin** [comment]
@mister-ben ,Can you show me the screenshot  with me.At https://songjuxin.github.io/videojs-exmpale/video-case.html

**Comment by mister-ben** [comment]
<img width="560" alt="image" src="https://user-images.githubusercontent.com/1676039/103352121-a13b2800-4aa5-11eb-94df-ed9f7f1a16b0.png">

**Comment by SongJuXin** [comment]
It's very strange.My three devices(two mac-chrome,one windows-chrome) occur the phenomenon.let me think again.

**Comment by SongJuXin** [comment]
Chrome has the problem.Firefox hasn't the problem.Can you show me a chrome screenshot with me?

This is my screenshot:
![image](https://user-images.githubusercontent.com/20874344/103401456-d8800680-4b83-11eb-907e-a42d46a1791a.png)

**Comment by gkatsev** [comment]
Your test page also includes
```css
    *{
      margin: 0;
      padding: 0;
    }
```
this could be causing issues here.

**Comment by friendtasy** [comment]
why i test https://songjuxin.github.io/videojs-exmpale/video-case.html this url,i feel progress is ok,but volumn is lower,and mouse move over can't make it move?

**Comment by friendtasy** [comment]
why i test https://songjuxin.github.io/videojs-exmpale/video-case.html this url,i feel progress is ok,but volumn is lower,and mouse move over can't make it move?

**Comment by SongJuXin** [comment]
At that time,due to time constraints,I used px instead of video-js's em  to solve this problem.

In fact, I found that the position was related to the font-size of Chrome browser less than 12px. When "font-size" of ". videos-js" element is modified to a larger value, such as 14px, the dot's position is correct.


My operating system is Chinese. 
Later, I found that when I switched the operating system to English, the dot's  position was also correct.


Provide the above ideas.
